BTCPayVsOthers.md

```diff
@@ -31,14 +31,15 @@ Every payment processor has features, here are some BTCPay features:
 * **Payment Buttons** - HTML embeddable donation and pay buttons.
 * **Unlimited Stores** - Merchants can process payments for their own stores, or for others.
 * **Translations** - Customers can pay in 20 different languages.
+* **Payment Requests** - Create & send an invoice requesting payment for goods or services.
```

Comment:
`Create & send a long-lived invoice requesting payment for goods or services. `?